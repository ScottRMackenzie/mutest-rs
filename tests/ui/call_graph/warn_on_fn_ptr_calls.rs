//@ print-targets
//@ stdout
//@ stderr

fn impl_call() {}

fn make_fn_ptr_call(f: fn()) {
    f();
}

#[test]
fn test() {
    make_fn_ptr_call(impl_call);
}
