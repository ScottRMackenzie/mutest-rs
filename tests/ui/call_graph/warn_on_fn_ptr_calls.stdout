tests -(0)-> make_fn_ptr_call at tests/ui/call_graph/warn_on_fn_ptr_calls.rs:7:1: 7:29 (#0)
  (0) test

targets: 1 total; 1 safe; 0 unsafe (0 tainted)
